<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sok-viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="valuesLabel">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="text">
      <string>Значения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QScrollArea" name="parametersScrollArea">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>149</width>
        <height>499</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="plotButtonsWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>631</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QGroupBox" name="zoomBox">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>0</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButtonXY">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>0</y>
         <width>49</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>49</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>X, Y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonX">
       <property name="geometry">
        <rect>
         <x>59</x>
         <y>0</y>
         <width>49</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>49</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonY">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>49</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>49</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonArea">
       <property name="geometry">
        <rect>
         <x>138</x>
         <y>0</y>
         <width>77</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Область</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="gridCheckBox">
      <property name="geometry">
       <rect>
        <x>347</x>
        <y>1</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Сетка</string>
      </property>
     </widget>
     <widget class="QLabel" name="backgroundLabel">
      <property name="geometry">
       <rect>
        <x>445</x>
        <y>2</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Фон</string>
      </property>
     </widget>
     <widget class="QLabel" name="scaleLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>2</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11px;</string>
      </property>
      <property name="text">
       <string>Режим</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="backgroundBox">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>0</y>
        <width>146</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="blackRadioButton">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>0</y>
         <width>74</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Черный</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="whiteRadioButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Белый</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="autozoomButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid;
	border-color: rgb(175, 175, 175);
	border-radius: 5px;
	background-color: rgb(241, 241, 241);
	font-size: 11px;
}
QPushButton::pressed {
	border: 1px solid;
	border-color: rgb(175, 175, 175);
	border-radius: 5px;
	background-color: rgb(185, 185, 185);
	font-size: 11px;
}</string>
      </property>
      <property name="text">
       <string>Авто</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="parametersLabel">
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="text">
      <string>Параметры</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="mainPlotWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>647</width>
       <height>501</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="plotLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QListWidget" name="valuesListWidget">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>501</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>161</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="controlBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="dataButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>70</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid;
	border-color: rgb(175, 175, 175);
	border-radius: 5px;
	background-color: rgb(241, 241, 241);
	font-size: 13px;
}
QPushButton::pressed {
	border: 1px solid;
	border-color: rgb(175, 175, 175);
	border-radius: 5px;
	background-color: rgb(185, 185, 185);
	font-size: 13px;
}</string>
      </property>
      <property name="text">
       <string>Файл</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="exportButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>3</y>
        <width>70</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid;
	border-color: rgb(175, 175, 175);
	border-radius: 5px;
	background-color: rgb(241, 241, 241);
	font-size: 13px;
}
QPushButton:pressed {
	border: 1px solid;
	border-color: rgb(175, 175, 175);
	border-radius: 5px;
	background-color: rgb(185, 185, 185);
	font-size: 13px;
}</string>
      </property>
      <property name="text">
       <string>Экспорт</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileStatusLabel">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>8</y>
        <width>165</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>165</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTabWidget" name="layersWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>3</y>
        <width>140</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>165</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab {
	background-color: rgb(241, 241, 241);
	border: 1px solid rgb(175, 175, 175);
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
	min-width: 20px;
	min-height: 20;
	padding: 1px;
	margin-right:2px;
}
QTabBar::tab:selected {
	background-color: rgb(185, 185, 185);
	border-color: rgb(175, 175, 175);
}
QTabWidget::pane{
	border-bottom: 0px;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="layer1">
       <attribute name="title">
        <string>1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="layer2">
       <attribute name="title">
        <string>2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="layer3">
       <attribute name="title">
        <string>3</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="layer4">
       <attribute name="title">
        <string>4</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="layer5">
       <attribute name="title">
        <string>5</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dataButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>dataButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonXY</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>zoomModeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonX</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>zoomModeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonY</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>zoomModeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>backgroundChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>backgroundChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autozoomButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>autozoomClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>gridClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>exportButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonArea</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>zoomModeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layersWidget</sender>
   <signal>tabBarClicked(int)</signal>
   <receiver>Widget</receiver>
   <slot>layerChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dataButtonClicked()</slot>
  <slot>zoomModeChanged()</slot>
  <slot>backgroundChanged()</slot>
  <slot>autozoomClicked()</slot>
  <slot>gridClicked()</slot>
  <slot>exportButtonClicked()</slot>
  <slot>layerChanged()</slot>
 </slots>
</ui>
